src/components/modules/ui.ts

```diff
@@ -620,7 +620,7 @@ export default class UI extends Module<UINodes> {
      * Do not fire check on clicks at the Inline Toolbar buttons
      */
     const target = event.target as HTMLElement;
-    const clickedInsideOfEditor = this.nodes.holder.contains(target) || Selection.isAtEditor;
```

Comment:
@neSpecc, I just tried to write some tests and encountered an issue with the `ui.ts:documentClicked` function:
![image](https://github.com/codex-team/editor.js/assets/52427793/d62b7653-389b-497a-b87b-134cbd2876fb)
It appears that this check causes clicks made by Cypress `click()` to behave differently from user-generated clicks (because Cypress-generated clicks will always have isTrusted == false). I'm concerned that this might break other tests.

Reply:
@neSpecc, I've commented this out with an explanation in a comment. Seems like we're not triggering Click events anywhere in the code so it shouldn't break anything.

Reply:
I've found a bug with Block Tunes pressing by Enter. It sometimes removes the block itself (like Enter pressed when block is selected). It can be reproduces in the `next` so I'm gonna dig in it soon. 

They can be related

Reply:
Hey @VikhorKonstantin. I've resolved several problems with Enter press. Please, pull the `next` and check if the problem still actual